config/suppressions-xpath.xml

```diff
@@ -19,4 +19,7 @@
     <suppress-xpath checks="CyclomaticComplexity"
                     query="/CLASS_DEF[./IDENT[@text='JavadocMethodCheck'
                     or @text='JavadocStyleCheck' or @text='CustomImportOrderCheck']]//METHOD_DEF"/>
+
+    <suppress-xpath checks="AbstractClassName"
```

Comment:
we usually add comment over suppression to explain reason.
this should be `Name of Test class should match name of target class`

Reply:
@pbludov , fyi.

Reply:
done